forms-html/zainab_dunia/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+</head>
+
+<body>
+  <form action="">
+    <label for="email">Enter your Email:</label><br>
+    <input type="email" id="email"><br>
+    <label for="name">Username:</label><br>
+    <input type="text" id="name"><br>
+    <label for="pass">Enter your password:</label><br>
+    <input type="password" id="pass"><br>
+    <label for="num">Quantity(between 1 and 5):</label>
+    <input type="number" id="num"><br>
+    <label for="num">Volume(between 0 and 50):</label><br>
+    <input type="range" id="num"><br>
+
+
+    <p>radio</p>
+    <input type="radio" id="male" name="gender" value="male">
+    <label for="male">Male</label><br>
+    <input type="radio" id="female" name="gender" value="female">
+    <label for="female">Female</label><br>
+    <input type="radio" id="other" name="gender" value="other">
+    <label for="other">Other</label></br><br>
+    <label for="check">CheckBox</label><br><br>
+    <input type="checkbox" id="vehicle1" name="vehicle1" value="Bike">
+    <label for="vehicle1"> I have a bike</label><br>
+    <input type="checkbox" id="vehicle2" name="vehicle2" value="Car">
+    <label for="vehicle2"> I have a car</label><br>
+    <input type="checkbox" id="vehicle3" name="vehicle3" value="Boat">
+    <label for="vehicle3"> I have a boat</label><br><br>
+
+
+
+
+    <label for="col">button</label><br><br>
+    <button type="button">Click Me!</button><br>
+    <lable for="color">select you favorite color</label>
+      <input type="color" id="color"><br>
+      <lable for="age">enter birthday date<label><br>
+          <input type="date" id="age"><br>
+          <label for="birthday">birthday (date and time)</label>
+
+          <input type="datetime" id="birthday"><br>
```

Comment:
The requirement was to all dates before 1980-1-1. You can do this using the max attribute in the input tag.